CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Support for adjusting the time that the Gateway Server schedules class C messages in advance per gateway.
+  - This requires a database migration (`ttn-lw-stack is-db migrate`) because of the added columns.
```

Comment:
```suggestion
- This requires a database migration (`ttn-lw-stack is-db migrate`) because of the added columns.
```

Reply:
Probably just join the lines

Reply:
There is an entry in the 3.3.0 version changelist with this format, which is why I used it. Can change if needed, but perhaps consistency would be better?

Reply:
OK fine with me too